### PR TITLE
Socint 258 configure eq toke for poc

### PR DESCRIPTION
**Pull request by SteveScorfield**
# Motivation and Context
It is required as part of the upcoming POC show and tell.

# What has changed
eq.py along with tests have been updated to accept the new values required.

# Links
https://collaborate2.ons.gov.uk/jira/browse/SOCINT-258

### CODE REVIEW

**Review by Paul-Joel — APPROVED** [review]

File: app/eq.py [hunk 1]
```diff
@@ -87,31 +98,33 @@ async def build(self):
...
-            'display_address':
-            self.build_display_address(self._sample_attributes),
+            'display_address': self.build_display_address(self._sample_attributes),
             'response_id': self._response_id,
             'account_service_url': self._account_service_url,
-            'account_service_log_out_url':
-            self._account_service_log_out_url,  # required for save/continue
+            'account_service_log_out_url': self._account_service_log_out_url,
             'channel': self._channel,
             'user_id': self._user_id,
```
Comment (rpcatling) [inline app/eq.py:116]: JIRA asks for `user_id` to be removed

File: app/eq.py [hunk 2]
```diff
@@ -87,31 +98,33 @@ async def build(self):
...
-            _tx_id,  # not required by eQ (will generate if does not exist)
+            'jti': str(uuid4()),
+            'tx_id': self._tx_id,
             'iat': int(time.time()),
-            'exp': int(time.time() +
-                       (5 * 60)),  # required by eQ for creating a new claim
-            'collection_exercise_sid': self._collex_id,  # required by eQ
+            'exp': int(time.time() + (5 * 60)),
+            'collection_exercise_sid': self._collex_id,
             'region_code': self.convert_region_code(self._region),
```
Comment (rpcatling) [inline app/eq.py:106]: although the JIRA asks for `region_code` to be hardcoded, I guess it is reasonable to leave as-is ?

File: app/eq.py [hunk 3]
```diff
@@ -87,31 +98,33 @@ async def build(self):
...
-            'ru_ref': self._uprn,  # new payload requires uprn to be ru_ref
-            'case_id':
-            self._case_id,  # not required by eQ but useful for downstream
+            # 'ru_ref': self._uprn,
+            'ru_ref': self._questionnaire_id,  # SOCINT-258 - temporary for use with POC
+            'case_id': self._case_id,
             'language_code': self._language_code,
```
Comment (rpcatling) [inline app/eq.py:110]: although the JIRA asks for region_code to be hardcoded, I guess it is reasonable to leave as-is ?

File: app/eq.py [hunk 4]
```diff
@@ -87,31 +98,33 @@ async def build(self):
...
-            'display_address':
-            self.build_display_address(self._sample_attributes),
+            'display_address': self.build_display_address(self._sample_attributes),
             'response_id': self._response_id,
             'account_service_url': self._account_service_url,
-            'account_service_log_out_url':
-            self._account_service_log_out_url,  # required for save/continue
+            'account_service_log_out_url': self._account_service_log_out_url,
             'channel': self._channel,
```
Comment (rpcatling) [inline app/eq.py:115]: JIRA asks for "RH" but we are providing "rh".  Is that OK ? Needs confirmation.

**Review by rpcatling — CHANGES_REQUESTED** [review]
some comments

File: app/eq.py (hunk 4 above)
Comment (SteveScorfield) [reply app/eq.py:115]: Already discussed with Paul and Sam. It needs to be 'rh'.

File: app/eq.py (hunk 3 above)
Comment (SteveScorfield) [reply app/eq.py:110]: As above

File: app/eq.py (hunk 1 above)
Comment (SteveScorfield) [reply app/eq.py:116]: This has been crossed out in the ticket and took it to mean that this needs to stay. As in it said to be removed but then crossed out.

File: app/eq.py (hunk 4 above)
Comment (rpcatling) [reply app/eq.py:115]: OK

File: app/eq.py (hunk 1 above)
Comment (SteveScorfield) [reply app/eq.py]: As agreed, user_id has been removed.

**Review by rpcatling — APPROVED** [review]

**Pull request merged**